java/client/src/main/java/glide/api/commands/VectorSearchBaseCommands.java

```diff
@@ -30,4 +32,34 @@ public interface VectorSearchBaseCommands {
      */
     CompletableFuture<String> ftcreate(
             String indexName, IndexType indexType, String[] prefixes, FieldInfo[] fields);
+
+    /**
+     * Uses the provided query expression to locate keys within an index. Once located, the count
+     * and/or content of indexed fields within those keys can be returned.
+     *
+     * @see TODO
```

Comment:
I guess this is because there's no Valkey docs for this yet?

Reply:
Right, will replace with a link when module become open source.